data_write/data_write.py

```diff
@@ -1002,9 +1074,10 @@ def write_tx_data():
                 parameters['main_antenna_count'] = np.uint32(len(rx_freq.rx_main_antennas))
                 parameters['intf_antenna_count'] = np.uint32(len(rx_freq.rx_intf_antennas))
                 parameters['freq'] = np.uint32(rx_freq.rxfreq)
-                parameters['comment'] = rx_freq.comment
+                parameters['rx_centre_freq'] = integration_meta.rx_centre_freq
```

Comment:
I think we should revisit this british vs us spelling thing. I do like being more canadian, but us english is way more common for people who have it as a second language, ans we do have lots of people who use our data from other countries.

Reply:
converted to US for in the data file.